Index: magics++-4.8.0/src/magics_config.h.in
===================================================================
--- magics++-4.8.0.orig/src/magics_config.h.in
+++ magics++-4.8.0/src/magics_config.h.in
@@ -71,7 +71,9 @@
 #cmakedefine MAGICS_COPYRIGHT_PERIOD
 
 /* Turn on the GeoTIFF output support */
+#ifndef HAVE_GEOTIFF
 #cmakedefine HAVE_GEOTIFF
+#endif
 
 /* Defines GIF output is supported. */
 #cmakedefine MAGICS_GIF
